django/apps/blue_management/blue_mgnt/views/policies.py

```diff
@@ -253,6 +253,10 @@ def _add_fields_from_preferences(self):
                 else:
                     inherit_choices = ROOT_INHERIT_CHOICES
 
+                if pref.name in ['DeletedItemsAutomaticPurge',
+                                 'HistoricalVersionAutomaticPurge']:
+                    inherit_choices = [x for x in inherit_choices if x[0] != '--unset--']
```

Comment:
Can we make the `inherit_field` a static text that says `Managed` greyed out if `len(inherit_choices) == 1`?